### PR TITLE
Feature - places autocomplete

### PR DESCRIPTION
**Pull request by joefitter**
* Allow setting of value directly, rather than looking up from list of objects

### CODE REVIEW

File: src/autocomplete/index.jsx [hunk 1]
```diff
@@ -19,6 +19,9 @@ export default function AutoComplete(props) {
       if (!val) {
         setValue('');
       }
+      if (props.stringValue) {
```
Comment (joefitter) [inline src/autocomplete/index.jsx]: feel free to suggest a better prop name for when options is a list of strings, not value/label pairs

File: src/autocomplete/index.jsx [hunk 2]
```diff
@@ -67,7 +75,12 @@ export default function AutoComplete(props) {
           inputValue: renderLabel,
           suggestion: renderLabel
         }}
-        onConfirm={option => setValue(option ? option.value : '')}
+        onConfirm={option => {
+          if (typeof option === 'string') {
+            return setValue(option);
+          }
+          setValue(option.value || '');
+        }}
```
Comment (wheelsandcogs) [inline src/autocomplete/index.jsx:83]: Would prefer this to be extracted now it's multi-line but not gonna die on that hill.

File: src/autocomplete/index.jsx [hunk 3]
```diff
@@ -67,7 +75,12 @@ export default function AutoComplete(props) {
           inputValue: renderLabel,
           suggestion: renderLabel
         }}
-        onConfirm={option => setValue(option ? option.value : '')}
+        onConfirm={option => {
+          if (typeof option === 'string') {
+            return setValue(option);
+          }
+          setValue(option.value || '');
```
Comment (joefitter) [inline src/autocomplete/index.jsx]:
```suggestion
          setValue(option ? option.value : '');
```

File: src/autocomplete/index.jsx (hunk 2 above)
Comment (joefitter) [reply src/autocomplete/index.jsx:83]: cheers for pointing out I'd actually changed how that works 😬

Comment (joefitter) [reply src/autocomplete/index.jsx:83]: reverted back to `option ?...`

**Pull request merged**